salt/orch/etcd-migrate.sls

```diff
@@ -95,3 +95,20 @@ update_mine:
       - salt: {{ worker_id }}-backup-etcd
 
 {% endfor %}
+
+{%- set admin = salt.saltutil.runner('mine.get', tgt='G@roles:admin', fun='network.ip_addrs', tgt_type='compound') %}
+{%- for admin_id, ip in admin.items() %}
+
+create-velum-pillar:
+  salt.function:
+    - tgt: {{ admin_id }}
+    - name: cmd.run
+    - arg:
+      - docker exec $(docker ps -a | grep velum-dashboard | awk '{print $1}') entrypoint.sh bundle exec rails runner 'ActiveRecord::Base.logger=nil; Pillar.create pillar: "api:etcd_version",  value: "etcd3"; Velum::Salt.orchestrate'
```

Comment:
Salt orchestration has problem with this command. I have to find other way of triggering it